src/index.js

```diff
@@ -1 +1,48 @@
 import './styles/style.css';
+import getData from './gameServer/serverget.js';
+import postData from './gameServer/serverpost.js';
+
+const refresh = document.getElementById('btn');
+
+const name = document.getElementById('Name');
+const score = document.getElementById('Score');
+
+const submit = document.querySelector('.btnsubmit');
+
+// eslint-disable-next-line operator-linebreak
+const url =
+  'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/4VEGe50cX5o9oalxLylU/scores/';
+
+const display = (data) => {
+  const list = document.querySelector('.list-item');
+
+  if (Array.isArray(data) && list) {
+    data.forEach((item) => {
+      const li = document.createElement('li');
+      li.innerHTML = `${item.user}: ${item.score}`;
+      list.appendChild(li);
+    });
+  }
+};
+
+refresh.addEventListener('click', async () => {
+  const data = await getData(url);
+
+  display(data);
+});
```

Comment:
- [ ] The functionality of displaying the data works, but if the user keeps clicking on the 'refresh' button, the data stacks, and then the page displays repeated data. Kindly fix this bug, so when the user clicks on the 'refresh' button, the data is displayed only once, not repeated data.
 - <img width="895" alt="image" src="https://user-images.githubusercontent.com/61919617/210641423-600dad33-c61d-47c2-81c0-6de014713f05.png">